core/src/main/java/org/apache/carbondata/core/scan/filter/executer/RowLevelRangeGrtThanFiterExecuterImpl.java

```diff
@@ -88,6 +88,13 @@ private void ifDefaultValueMatchesFilter() {
   private boolean isScanRequired(byte[] blockMaxValue, byte[][] filterValues) {
     boolean isScanRequired = false;
     if (isDimensionPresentInCurrentBlock[0]) {
+      // if a column is unsorted, min and max for that column will not be correct and hence
```

Comment:
During data loading even if data is not sorted we are calculating the min max so not need to add this code, in all the filter classes only if inverted index is false we need to go iterative approach. Please check in all the rowlevel range filter